showindividual.php

```diff
@@ -1,23 +1,79 @@
-<?php ?>
+<?php
+    require_once('functions.php');
+    $db = new PDO('mysql:host=db; dbname=collections-project2', 'root', 'password');
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    $allItems = $db->prepare("SELECT * FROM `collections`;");
+    $allItems->execute();
+    $allResults = $allItems->fetchAll();
+    $numberOfItemInArray = count($allResults);
+    $lastItem = $allResults[$numberOfItemInArray - 1];
```

Comment:
Have no idea, but likely that there is a predefined function for this in php. If not then clever work around.

Reply:
function in functions page ahaha